### DISCUSSION TITLE
Help with testing - Enroute vs Flight Simulators and Traffic Receivers

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#311] [Team Posts]
Dear all,

the code used to connect to flight simulators and traffic receivers has been greatly expanded, with the obvious potential for introducing new errors. I would therefore like to ask for your help with testing the latest beta version (=2.6.9).

- If you own a flight simulator program, could you please check that the app connects to the flight simulator and shows the data as expected? I do not have a Windows machine, so I cannot test MS Flight Simulator, which is however arguably the most important program around.
- Dito for the traffic receiver that you have in your airplane. The app should now work with almost all devices on the market (FLARM, SkyEcho, …) but a lot of testing needs to be done.

Thanks again for your support!

Stefan.

**Comment by Ivan081** [comment]
If you are interested I have Flight Simulator 2020 and a windows 10 PC.

Can you explain to me what I have to do?

PS. Remember that I'm busy with flight exams this weekend


Il giorno mer 5 mag 2021 alle ore 08:01 Stefan Kebekus <
***@***.***> ha scritto:

**Comment by kebekus** [comment]
Sorry for the late reply. The answer is: I do not know exactly. At

-  https://akaflieg-freiburg.github.io/enrouteText/experimentalmanual/html/02-steps/simulator.html

you find how to connect to the X-Plane 11 simulator software; I would need something similar for the MS Flight Simulator. One explanation how this might work is here:

- https://support.foreflight.com/hc/en-us/articles/204115275-How-do-I-connect-Microsoft-Flight-Simulator-FS-X-or-FS-2004-to-ForeFlight-

Best -- Stefan.
